src/System.Management.Automation/resources/Metadata.resx

```diff
@@ -123,6 +123,18 @@
   <data name="ValidateRangeElementType" xml:space="preserve">
     <value>The argument cannot be validated because its type "{0}" is not the same type ({1}) as the maximum and minimum limits of the parameter. Make sure the argument is of type {1} and then try the command again.</value>
   </data>
+  <data name="ValidateRangePositiveFailure" xml:space="preserve">
+    <value>The argument "{0}" cannot be accepted because its value is not greater than zero.</value>
```

Comment:
For consistency, validation errors use `The argument "{0} cannot be validated` - the `cannot be accepted` phrase is not commonly used in our code base, basically just in this resx file, and maybe we should change those messages.